statistics-production/scrums.qmd

```diff
@@ -41,16 +41,12 @@ If you are responsible for signing off publications, then please download and se
 
-<div align=center>
-<iframe width="640" height="360" src="https://web.microsoftstream.com/embed/video/73fa0d31-b8b3-4889-bfcf-c76c33c1fe87?autoplay=false&showinfo=false" allowfullscreen style="border:none;"></iframe>
-</div>
+A full before / after scrum along is available to [watch](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a), with slides available to [download separately](../resources/Content_Design_Scrum_Progression.pptx), upon request.
 
-A full before / after scrum along is available to [watch](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a), with slides available to [download separately](../resources/Content_Design_Scrum_Progression.pptx).
+For a walk through of some of the end to end benefits the scrums have had, you may ask for access to this [video for Schools and pupils Statistics Team](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a).
 
-For a walk through of some of the end to end benefits the scrums have had, take a look at this [video for Schools and pupils Statistics Team](https://educationgovuk-my.sharepoint.com/:v:/r/personal/heather_brown_education_gov_uk/Documents/Recordings/Content%20Design%20Scrum_%20Progression%20Measures%20Statistics-20210628_123245-Meeting%20Recording.mp4?csf=1&web=1&e=xc032a).
-
-Finally, an example of a team who has been through the scrum process for 3 publications talk through their realised benefits, showcasing the type of benefits potentially others could also realise is also available to [watch on sharepoint](https://educationgovuk-my.sharepoint.com/:v:/g/personal/heather_brown_education_gov_uk/EQx_j1oEh0BMqLbMLtnf2ikB4FJIBfEjKptqIDUkoZVgvg?e=a0xHNJ).
+Finally, an example of a team who has been through the scrum process for 3 publications talk through their realised benefits, showcasing the type of benefits potentially others could also realise is also available on [demand on sharepoint](https://educationgovuk-my.sharepoint.com/:v:/g/personal/heather_brown_education_gov_uk/EQx_j1oEh0BMqLbMLtnf2ikB4FJIBfEjKptqIDUkoZVgvg?e=a0xHNJ).
```

Comment:
Think the same for this one. Given Heather's left the dept, we're probably just going to have to remove this link entirely. I can access the video at least, but I can't export it to anywhere more accessible.